app/lib/postcodeValidator.js

```diff
@@ -5,6 +5,7 @@ const log = require('../lib/logger');
 function setInvalidPostcodeLabel(res, postcode) {
   // eslint-disable-next-line no-param-reassign
   res.locals.searchErrorLabel = `The postcode '${postcode}' does not exist.`;
+  res.locals.searchErrorClass = 'pc';
```

Comment:
Sorry to be picky but this would be better as 'postcode' take make intent clearer and given that we already have `form-group__postcode` and `error__search`.